### ISSUE TITLE
Articles should relate to instutions

### ISSUE THREAD

**Issue by TheBeege** [TheBeege/SciNet#8]
In the current data model, the only way to relate an article to an institution is through the author. In the event we don't have historical institutions for an author, this could lead to mislabeling the institution an article was published under.  While relating articles to institutions seems a little redundant, it prevents potential misrepresentation or data loss. 

I created this enhancement because I was originally posing it as a question. After writing this, I realized creating the relationship between articles and institutions is the correct way to go. 

I'll make this a many-to-many relationship to support collaborative efforts.

**Comment by anthonyselkowitz** [comment]
I agree about the articles to institutions connection. The authors to I institutions could be useful for tracking if the author changes institutions frequently.
